lib/reek/ast/object_refs.rb

```diff
@@ -1,33 +1,30 @@
 module Reek
-  module AST
+  module AST  # :nodoc:
```

Comment:
What is this `# :nodoc:` supposed to accomplish? I don't see it having any effect on YARD's output.